awx/ui_next/src/components/MultiSelect/MultiSelect.jsx

```diff
@@ -106,16 +106,15 @@ class MultiSelect extends Component {
     const { input, chipItems } = this.state;
     const { onAddNewItem } = this.props;
     const newChip = { name: input, id: Math.random() };
-    if (event.key !== 'Tab') {
-      return;
+    if (event.key === 'Enter') {
+      event.preventDefault();
```

Comment:
:+1: nice, I think using `preventDefault` for `Enter` is a good way to handle this.